quinn/src/streams.rs

```diff
@@ -479,46 +518,88 @@ where
         self.stream
     }
 
+    /// Handle common logic related to reading out of a receive stream
+    ///
+    /// This takes an `FnMut` closure that takes care of the actual reading process, matching
+    /// the detailed read semantics for the calling function with a particular return type.
+    /// The closure can read from the passed `&mut Chunks` and has to return a result in two parts:
+    ///
+    /// * Amount of data read, in the type expected by the calling read method
+    /// * Status after the final read, with the `Ok` value denoting whether the stream has finished
```

Comment:
Description of the return value needs to be updated

Reply:
Done.